syntax = "proto2";
package PluginProtocolProto;
import "PluginAPIMessage.proto";
message Wdctl {
  repeated string commands = 1;
}

message WdService{
  required PluginAPIMessage apimessage = 1;
}

message WatchdogMsg{
  optional Wdctl wdctl = 1;
  optional WdService wdservice = 2;
}

message WatchdogMsgs {
  repeated WatchdogMsg messages = 1;
}